apps/dashboard/src/lib/components/Question/constants.ts

```diff
@@ -22,6 +22,8 @@ export const QUESTION_TYPES = [
 export const QUESTION_TEMPLATE = {
   id: 1,
   title: '',
+  hint: '',
+  explanation: '',
```

Comment:
What's the difference between hint and explanation?

Reply:
you said the users should have a hint then after they get the question right, "if" the question has an explanation, the explanation should then show before moving to the next question.
"Show a correct answer hint in the answers. That is, after they selected the answer, they get a message explaining the answer further."

Reply:
@tunny17 Okay a miscommunication on my end. The hint wasn't something I had initially thought about, I just meant that the teacher enters an explanation to the question and at when the students get the question we then show them the explanation before moving to the next question

However I am curious how you implemented this hint, please share a video